docs/theme/mlflow/static/js/custom.js

```diff
@@ -154,77 +154,6 @@ require=(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof requ
 
 // CUSTOM JS
 
-// feedback link
```

Comment:
This JS was setting an element on the doc that has a conflict with how nbsphinx renders notebooks, creating a class path collision (verified in dev console). 
Since 2.8 docs overhaul replaced this custom clipboard functionality with a simpler approach that copies the contents of a text div containing code formatting, this is useless to keep in the repo.